frontends/web/src/components/headerssync/headerssync.tsx

```diff
@@ -16,89 +16,64 @@
  * limitations under the License.
  */
 
-import { Component} from 'react';
+import { FunctionComponent, useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
 import { CoinCode } from '../../api/account';
-import { subscribe } from '../../decorators/subscribe';
-import { translate, TranslateProps } from '../../decorators/translate';
+import { subscribeCoinHeaders } from '../../api/coins';
+import { useSubscribe } from '../../hooks/api';
 import Spinner from '../spinner/ascii';
 import style from './headerssync.module.css';
 
-interface IHeadersSyncProps {
+interface Props {
     coinCode: CoinCode;
 }
 
-interface IStatus {
-    targetHeight: number;
-    tip: number;
-    tipAtInitTime: number;
-    tipHashHex: string;
+const useMountedRef = () => {
```

Comment:
move this to a place where it can be reused? you use the same trick three more times in this PR, all of them could use `useMountedRef()` instead, right?

Reply:
I moved it to `hooks/utils.ts` :+1: